react-charts/line-chart/step5.md

```diff
@@ -1,36 +1,56 @@
-Now we'll add tooltips to the line chart...
+Add axis labels to the line chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the code for the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It should look like this:
 
 <pre class="file">
 &lt;Chart
+  constrainToVisibleArea
+  containerComponent={&lt;ChartVoronoiContainer labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`} /&gt;}
   maxDomain={{y: 10}}
   minDomain={{y: 0}}
   height={250}
   width={600}
 &gt;
 </pre>
 
-3) Add the following property to that section:
+3) <strong>Add the `padding` property to that section:</strong>
+
+This will add more padding to the bottom and left properties to accommodate the axis labels.
+
+Copy this code to the editor:
 
 <pre class="file" data-target="clipboard">
-containerComponent={
-  &lt;ChartVoronoiContainer
-    constrainToVisibleArea
-    labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`}
-  /&gt;
-}
+padding={{
+  bottom: 75,
+  left: 75,
+  right: 50,
+  top: 50
+}}
+</pre>
+
+4) <strong>Locate the code between the `<Chart>` and `<ChartGroup>`.</strong>
+
+<pre class="file">
+&lt;ChartGroup&gt;
+...
+&lt;/ChartGroup&gt;
 </pre>
 
-- The `constrainToVisibleArea` property will alter the position of the tooltip so that it exactly fits within the rendered SVG
-- The `containerComponent` property takes a component instance which will be used to create a container element for standalone charts
-- The `ChartVoronoiContainer` is used to create voronoi overlays for charts, which are useful for attaching events to data points, such as tooltips
+5) <strong>Between these components add the `<ChartAxis>` components.</strong>

```

Comment:
```suggestion
5) <strong>Between these components add the `<ChartAxis>` components.</strong>
The `<Chart>` component displays an axis by default. Add labels and gridlines here to enhance the component.
a) <strong>Add the `<ChartAxis>` component with a label property.</strong>

It should look like this:

<pre class="file" data-target="clipboard">
&lt;ChartAxis label=&quot;Years&quot;/&gt;
</pre>

b) <strong>Add another `<ChartAxis>` component with multiple properties.</strong>

The `dependentAxis` property specifies whether the axis corresponds to dependent data points (e.g., usually the y-axis).
```

Reply:
I updated this section as Christie prescribed above.